website/content/en/v0.5.5/getting-started/_index.md

```diff
@@ -142,7 +142,7 @@ eksctl. Thus, we don't need the helm chart to do that.
 helm repo add karpenter https://charts.karpenter.sh
 helm repo update
 helm upgrade --install karpenter karpenter/karpenter --namespace karpenter \
-  --create-namespace --set serviceAccount.create=false --version v0.5.5 \
```

Comment:
IIUC this should include the IRSA annotations if we remove this, or else it will override the service account created above with eksctl?